.github/workflows/tests.yml

```
@@ -23,7 +23,7 @@ jobs:
     name: Unix
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-latest, macos-13]
```

We could also add [`ubuntu-22.04`](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md) which is used by [Homebrew's Linux job](https://github.com/Homebrew/homebrew-core/actions/runs/11708308681/job/32609917766?pr=195371#step:1:8).

```suggestion
        os: [ubuntu-latest, ubuntu-22.04, macos-latest, macos-13]
```

---

This is currently ubuntu-latest. They tried to move to ubuntu-24.04 but had to roll it back, I think it moves Jan 2025 now.